etc/templates/profile.template

```diff
@@ -21,6 +44,13 @@
 #noblacklist /usr/local/lib/python2*
 #noblacklist /usr/local/lib/python3*
 
+# Allow perl (blacklisted by disable-interpreters.inc)
```

Comment:
more generally, maybe we should break these out into their own so they can be included
```
include allow-python.inc
include allow-perl.inc
```
thoughts?

Reply:
Looks neat. I will be happy to add that once we have such includes.